share/spice/quandl/world_bank/world_bank_primary.yml

```diff
@@ -0,0 +1,22 @@
+population:	SP_POP_TOTL
+population female:	SP_POP_TOTL_FE_ZS
+population is female:	SP_POP_TOTL_FE_ZS
+female population:	SP_POP_TOTL_FE_ZS
+gross external debt:	DT_DOD_DECT_CD_FD
+education expenditure :	NY_ADJ_AEDU_CD
+debt sector:	DP_DOD_DECT_CR_GG_CD
+gross national expenditure:	NE_DAB_TOTL_CN
+stocks traded:	CM_MKT_TRAD_CD
+urban population:	SP_URB_TOTL
+labor force:	SL_TLF_TOTL_IN
+food production:	AG_PRD_FOOD_XD
+telephone lines:	IT_MLT_MAIN
+national expenditure :	NE_DAB_TOTL_CN
+shipping connectivity:	IS_SHP_GCNW_XQ
+shipping connectivity index:	IS_SHP_GCNW_XQ
+shipping index: IS_SHP_GCNW_XQ
+droughts:	EN_CLC_MDAT_ZS
```

Comment:
Done!  Note that this often returns 0% change (e.g. "Argentina rainfall").  Just having "0%" looks odd.  What do you think about adding the word "change" after the percent change?